### PR TITLE
Simplify generating culture names

### PR DESCRIPTION
**Pull request by hishamco**
Simplify generating culture names using simple LINQ query with newly format style. 

For better performance we can fetch the culture names rather than fetching the entire ```CultureInfo``` list for all the cultures

### CODE REVIEW

File: src/CultureInfoGenerator/Program.cs [hunk 1]
```diff
@@ -58,24 +59,7 @@ public static partial class CultureInfoCache
...
-                    {
-                        writer.WriteLine(",");
-                    }
-                    else
-                    {
-                        // Last entry
-                        writer.WriteLine();
-                    }
-                }
+                writer.WriteLine(string.Join(",\n", cultures.Select(c => $"            \"{c.Name}\"")));
```
Comment (Eilon) [inline src/CultureInfoGenerator/Program.cs]: `\n` should probably be `Environment.NewLine`

Comment (hishamco) [inline src/CultureInfoGenerator/Program.cs]: I do that to avoid using concatenation, but I can use string interpolation :)

Comment (DamianEdwards) [inline src/CultureInfoGenerator/Program.cs]: I wouldn't worry about concatenation, this is a build-time tool we'll run once per Windows OS release. Optimize for code clarity, if anything at all.

Comment (Eilon) [inline src/CultureInfoGenerator/Program.cs]: I think it's reasonable to argue that using `string.Join` is clearer than a `for()` loop.

Comment (DamianEdwards) [inline src/CultureInfoGenerator/Program.cs]: Oh totally, not disputing that.

**Comment by hishamco** [comment]
The PR has been update

**Comment by Eilon** [comment]
@kirthik please review; the code looks good to me. It's just one of the build tools.

**Comment by kirthik** [comment]
Looks good to me. I will merge it.

**Comment by kirthik** [comment]
merged 59d957df3193902c9ee9beea0fe488b75fecee46

**Pull request closed without merging**